app/controllers/welcome_controller.rb

```diff
@@ -1,6 +1,7 @@
 class WelcomeController < ApplicationController
   def index
     @videos = Episode.active.limit(6).order('published_at DESC')
+    get_events
```

Comment:
Since we're explicitly calling this here, it would be nice to do the assignment here as well, so we can easily see what's passed to the controller. :grinning: 

```
@events = get_events
```

and change the getter to just return the results:

```
def get_events
  WebuildEventsService.new.get_events
end
```